docs/explanation/security-overview.md

```diff
@@ -0,0 +1,38 @@
+# WordPress security overview
+
+This explanation covers several security-related topics for the WordPress charm.
+
+## Secrets
+
+Secrets required to operate the WordPress application are generated by the WordPress
+charm using the Python's secure `secrets` standard library. The following fields in the WordPress configuration are generated by the charm:
+- auth_key
+- secure_auth_key
+- logged_in_key
+- nonce_key
+- auth_salt
+- secure_auth_salt
+- logged_in_salt
+- nonce_salt
+
+## Updates
+
+For security reasons, the WordPress charm does not allow updates of the WordPress application itself automatically.
```

Comment:
WordPress cannot be upgraded from the web administration page primarily because it is containerized, so restarting the container reverts any changes made by the upgrade. And if there are multiple units, upgrading from the web administration page causes inconsistencies between units. The same applies to plugin and theme installation or upgrades.